### PR TITLE
fuckin too many t-s

### PR DESCRIPTION
**Pull request by valeriahacks**
removed some bitch ass T-s. See title for more.

### CODE REVIEW

**Review by kirillito — APPROVED** [review]

**Comment by graypegg** [comment]
Added help wanted tag, needs input from @tgunness

**Comment by tgunness** [comment]
`print('hello ${i + 1}' + bitttch('VIB'));`

hard coded keys/string are just wrong. please fix this. 

there is also a grammatical error in your print statement within the for loop - missing space.

File: dart.dart [hunk 1]
```diff
@@ -1,9 +1,9 @@
 void main() {
   for (int i = 0; i < 5; i++) {
-    print('hello ${i + 1}' + bittttch('VIB'));
+    print('hello ${i + 1}' + bitttch('VIB'));
   }
 }
 
-String bittttch (name) {
+String bitttch (name) {
```
Comment (tgunness) [inline dart.dart:7]: missing the Type on your param

and you can make this a one liner. 

`String bittch(String name) => 'yo bitch ' + name;`

**Review by tgunness — CHANGES_REQUESTED** [review]

**Comment by tgunness** [comment]
@kirillito i am very disappointed this was approved!

File: dart.dart (hunk 1 above)
Comment (kirillito) [reply dart.dart:7]: One liners are not always a good idea

**Review by kirillito — CHANGES_REQUESTED** [review]
Trishan is disappointed

**Comment by valeriahacks** [comment]
lgtm @tgunness

**Pull request merged**